bazel_example/gapic-generator-cloud/BUILD.bazel

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Defines the targets for the google cloud language library to be built
+with gapic-generator-ruby cloud flavor
+"""
+load("@gapic_generator_ruby//rules_ruby_gapic:ruby_gapic.bzl", 
+  "ruby_gapic_cloud_library",
+  "ruby_proto_library",
+  "ruby_grpc_library")
+
+load("@gapic_generator_ruby//rules_ruby_gapic:ruby_gapic_pkg.bzl", 
+  "ruby_gapic_assembly_pkg")
+
+##
+# completed google-cloud language library combined from
+# * protoc ruby output
+# * grpc-ruby protoc plugin output
+# * gapic-generator-ruby cloud flavor output
+#
+ruby_gapic_assembly_pkg(
+  name = "google-cloud-language-v1-ruby",
+  deps = [
+    ":language_ruby_gapic",
+    ":language_ruby_grpc",
+    ":language_ruby_proto",
+  ],
+)
+
+##
+# just the gapic-generator-ruby cloud flavor output
+#
+ruby_gapic_cloud_library(
+  name = "language_ruby_gapic",
+  srcs = ["@com_google_googleapis//google/cloud/language/v1:language_proto_with_info"],
+  ruby_cloud_params = {
+    ":gem.:name": "google-cloud-language-v1",
+    ":gem.:title": "Natural Language V1",
+    ":gem.:description": "Provides natural language understanding technologies\\, such as sentiment analysis\\, entity recognition\\, entity sentiment analysis\\, and other text annotations.",
```

Comment:
I thought we were going to remove the need to escape the commas.

Reply:
done